include/frozen/bits/exceptions.h

```diff
@@ -23,7 +23,7 @@
 #ifndef FROZEN_LETITGO_EXCEPTIONS_H
 #define FROZEN_LETITGO_EXCEPTIONS_H
 
-#if defined(FROZEN_NO_EXCEPTIONS) || (defined(_MSC_VER) && !defined(_CPPUNWIND)) || (defined(__cpp_exceptions) && __cpp_exceptions)
+#if defined(FROZEN_NO_EXCEPTIONS) || (defined(_MSC_VER) && !defined(_CPPUNWIND)) || (defined(__cpp_exceptions) && !__cpp_exceptions)
```

Comment:
You may as well skip the `defined` test:
```suggestion
#if defined(FROZEN_NO_EXCEPTIONS) || (defined(_MSC_VER) && !defined(_CPPUNWIND)) || !__cpp_exceptions
```

Because inside `#if` preprocessor conditionals undefined names are always implicitly interpreted as `0`.